src/Twig/Extension/UploaderExtension.php

```diff
@@ -41,6 +41,16 @@ public function getFunctions(): array
      */
     public function asset($object, ?string $fieldName = null, ?string $className = null): ?string
     {
+        if (!\is_object($object)) {
```

Comment:
This is already deprecated in the helper. If we keep it here too, user will get a duplicate deprecation warning, right?

Reply:
Only if you don't use a custom UploaderHelper.

Reply:
If you want to avoid duplicate deprecation, I can add a check `!$this->uploaderHelper instanceof UploadHelper`.
WDYT ?